README.md

```diff
@@ -13,9 +8,15 @@ and [RDS Enhanced Monitoring via CloudWatch Logs](https://docs.aws.amazon.com/Am
 Based on [Technofy/cloudwatch_exporter](https://github.com/Technofy/cloudwatch_exporter),
 but very little of the original code remained.
 
+Forked from [Percona RDS exporter](https://github.com/percona/rds_exporter). **Enhancements:**
+* Dockerfile was rewritten to multistage build
+* Added script `generate_rds_list.py` that creates list of RDS instances.  
+Script supports filtering: if `USE_FILTER` environment variable was set with `true` value, only metrics for instances with tag `UseExporter=true` will be listed. 
```

Comment:
The script supports filtering: if the environment variable `USE_FILTER` has the value `true`, only metrics for instances with the tag `UseExporter=true` will be listed.

Reply:
Fixed in https://github.com/pdffillerdocker/rds_exporter/commit/4e36931a0807b3e836b4f63cea3352a3ee36ed68